rll-examples/install-packages.sh

```diff
@@ -0,0 +1,114 @@
+#! /bin/bash
+# ---
+# RightScript Name: SYS packages install
+# Description: |
+#   Installs packages required by RightScripts.
+#   To handle naming variations, prefix package names with "yum:" or "apt:".
+# Inputs:
+#   PACKAGES_INSTALL:
+#     Input Type: single
+#     Category: System
+#     Description: |
+#       Set to "false" in order to skip combined package installation of RS_PACKAGES.
+#     Required: true
+#     Advanced: true
+#     Default: text:true
+#     Possible Values:
+#       - text:true
+#       - text:false
+# ...
```

Comment:
Perhaps an input to check if `update` by the package manager should be run?  Thinking that people may use this to update their packages?